tests/www/test_app.py

```diff
@@ -290,11 +288,12 @@ class TestFlaskCli:
     def test_flask_cli_should_display_routes(self, capsys):
         with mock.patch.dict("os.environ", FLASK_APP="airflow.www.app:cached_app"), mock.patch.object(
             sys, "argv", ["flask", "routes"]
-        ), pytest.raises(SystemExit):
+        ):
             from flask import __main__
 
             # We are not using run_module because of https://github.com/pytest-dev/pytest/issues/9007
-            runpy.run_path(__main__.__file__, run_name="main")
+            with pytest.raises(SystemExit):
+                runpy.run_path(__main__.__file__, run_name="main")
```

Comment:
I suspect that this test will fail and didn't reach `runpy.run_path` need to check what we actually check here and why